### DISCUSSION TITLE
Statistics Placeholder (%cmi_user_stats_[mainStat(:optionalSubStat)]%)

### DISCUSSION THREAD

**Discussion by Dusten9752** [Zrips/CMI#7880] [Enhancement]
I was wondering if this could be enhanced so that the placeholder is able to be parsed per player for a leaderboard for player kills (similar to how /baltop and /playtimetop are able to be parsed).

**Comment by CloudeLecaw** [comment]
thats not possible to have a leaderborad like this.
its would need to record the stats in cmi database to can build a leaderboard
but stats are taken from playersfiles stats folder (mainworld folder)
this would ment the plugin would need to real all stats of all players...
but you can use a leaderbord plugin to track selected stats and create a leaderboard with them.
it delects the defined staticis for leaderboard of all player that are online since you created the stat.

the plugin is called AJLeaderboards and can be taken here https://www.spigotmc.org/resources/ajleaderboards.85548/
